openhtf/output/callbacks/mfg_inspector.py

```diff
@@ -155,7 +155,7 @@ def _extract_attachments(phase, testrun, used_parameter_names):
     name = _ensure_unique_parameter_name(name, used_parameter_names)
     testrun_param = testrun.info_parameters.add()
     testrun_param.name = name
-    if isinstance(attachment_data, unicode):
+    if isinstance(attachment_data, str):
```

Comment:
Does this work as intended on Python2 now? 2to3 did this, with no concern for backwards compatibility.